planemo/xml/xsd/tool/citations.xsd

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I know how much fun <s>copy and pasting</s> scripted import of files from other projects is ;)